jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandlerTest.java

```diff
@@ -23,22 +23,24 @@
 
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
-import mockit.Mocked;
+import org.junit.Before;
 import org.junit.Test;
-
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.mock;
 
 /**
  * @author roland
  * @since 23.10.18
  */
 public class FromConfigRegistryAuthHandlerTest {
 
-  @Mocked
   private KitLogger log;
-
+  @Before
+  public void setUp() throws Exception {
+    log = mock(KitLogger.class);
```

Comment:
As mentioned in other tests, please use SilentLogger or a combination of Spy + SilentLogger (if we perform verifications on the log)